examples/closures/src/lib.rs

```diff
@@ -1,6 +1,4 @@
-extern crate js_sys;
-extern crate wasm_bindgen;
-extern crate web_sys;
+use web_sys;
```

Comment:
I believe in 2018 you can omit this too!

Reply:
Removed